### PR TITLE
Make advancements localizable

### PR DESCRIPTION
**Pull request by mcBegins2Snow**
Test success on screenshot
Language: Chinese

@zixuan-eb If you found problem, please comment to me.

### CODE REVIEW

File: src/main/resources/assets/atum/lang/zh_cn.lang [hunk 1]
```diff
@@ -1,911 +1,973 @@
...
+advancement.atum.ahumpintheroad.desc=驯服一只骆驼
+advancement.atum.ajourney=旅程
+advancement.atum.ajourney.desc=到访所有阿图姆的生物群系
+advancement.atum.amirage=海市蜃楼
+advancement.atum.amirage.desc=到访死去的绿洲
+advancement.atum.aquaculture=水产养殖
+advancement.atum.aquaculture.desc=在阿图姆捕捉一条鱼
+advancement.atum.assassinassin=Assassinassin
+advancement.atum.assassinassin.desc=刺杀一个刺客
+advancement.atum.atumed=Atumed
```
Comment (zixuan-eb) [inline src/main/resources/assets/atum/lang/zh_cn.lang]: 阿图姆化身

File: src/main/resources/assets/atum/lang/zh_cn.lang [hunk 2]
```diff
@@ -1,911 +1,973 @@
...
+advancement.atum.pyramidscheme.desc=进入金字塔
+advancement.atum.radyforaction=准备行动
+advancement.atum.radyforaction.desc=获得完整的拉套装
+advancement.atum.root=阿图姆
+advancement.atum.root.desc=开始你的阿图姆之旅
+advancement.atum.ruinedit=毁了它
+advancement.atum.ruinedit.desc=进入遗迹
+advancement.atum.sandyboi=Sandy Boi
+advancement.atum.sandyboi.desc=驯服一只沙漠狼
+advancement.atum.stonecarver=石头雕刻家
```
Comment (zixuan-eb) [inline src/main/resources/assets/atum/lang/zh_cn.lang]: 切石者

File: src/main/resources/assets/atum/lang/zh_cn.lang [hunk 3]
```diff
@@ -1,911 +1,973 @@
...
+item.atum.loot.dirty.idol.name=肮脏的神像
+item.atum.loot.dirty.necklace.name=肮脏的项链
+item.atum.loot.dirty.ring.name=肮脏的指环
+item.atum.loot.dirty.broach.name=肮脏的胸针
+item.atum.loot.dirty.scepter.name=肮脏的权杖
+item.atum.loot.silver.idol.name=淡灰色神像
+item.atum.loot.silver.necklace.name=淡灰色项链
+item.atum.loot.silver.ring.name=淡灰色指环
+item.atum.loot.silver.broach.name=淡灰色胸针
+item.atum.loot.silver.scepter.name=淡灰色权杖
```
Comment (zixuan-eb) [inline src/main/resources/assets/atum/lang/zh_cn.lang]: help change these 5 '淡灰色' to ‘银制’, and '骨质' to '骨制', '铁质' to '铁制', '金质' to '金制'， thankyou_(:зゝ∠)_

File: src/main/resources/assets/atum/lang/zh_cn.lang (hunk 1 above)
Comment (zixuan-eb) [reply src/main/resources/assets/atum/lang/zh_cn.lang]: 刺客刺杀者

**Comment by GirafiStudios** [comment]
Thanks a lot for the PR!
It was on my list of stuff I wanted to get fixed at some point, so I really appreciate you doing it ^^
Also thanks @zixuan-eb for giving feedback :)

**Pull request merged**